torii/pytests/common/settings.py

```diff
@@ -7,7 +7,7 @@
 BASE_DIR = os.path.dirname(
     os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 )
-CONFIG_DIR = os.path.join(BASE_DIR, "configs/swarm/client.toml")
+CONFIG_DIR = os.path.join(BASE_DIR, "defaults/client.toml")
```

Comment:
```suggestion
CONFIG_DIR = os.path.join(BASE_DIR, "default/client.toml")
```

I'd prefer singular?

Reply:
I like the plural version more, but I can't put my finger on why